app/javascript/pages/getAssistance/privacy.tsx

```diff
@@ -1,12 +1,133 @@
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.infoSharingTitle")}</h2>
+              <ul className="list-disc ml-7">
+                <li>{t("privacyPolicy.infoSharingP1")}</li>
+              </ul>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.analyticsTitle")}</h2>
+              <p>
+                {renderMarkup(
+                  `${t("privacyPolicy.analyticsP1", {
+                    termsLink:
+                      '<a href="http://www.google.com/analytics/terms/us.html" target="_blank">',
+                    privacyLink:
+                      '<a href="http://www.google.com/policies/privacy/" target="_blank">',
+                    optOutLink:
+                      '<a href="https://tools.google.com/dlpage/gaoptout?hl=en" target="_blank">',
+                    linkEnd: "</a>",
+                  })}`
+                )}
+              </p>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.linksTitle")}</h2>
+              <ul className="list-disc ml-7 space-y-2.5">
+                <li>{t("privacyPolicy.linksP1")}</li>
+                <li>{t("privacyPolicy.linksP2")}</li>
+                <li>{t("privacyPolicy.linksP3")}</li>
+              </ul>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.siteSecurityTitle")}</h2>
+              <ul className="list-disc ml-7 space-y-2.5">
+                <li>{t("privacyPolicy.siteSecurityP1")}</li>
+                <li>{t("privacyPolicy.siteSecurityP2")}</li>
+              </ul>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.policyChangesTitle")}</h2>
+              <ul className="list-disc ml-7">
+                <li>{t("privacyPolicy.policyChangesP1")}</li>
+              </ul>
+            </div>
+            <div className="md:pr-11 md:pl-0">
+              <hr />
+            </div>
+            <div className="space-y-4 p-6 md:py-11 md:pr-11 lg:pl-0">
+              <h2>{t("privacyPolicy.questionsTitle")}</h2>
+              <p>
+                {renderMarkup(
+                  `${t("privacyPolicy.questionsP1", {
+                    telLink: '<a href="tel:4157015500">415-701-5500</a>',
+                    emailLink:
+                      '<a href="mailto:sfhousinginfo@sfgov.org">sfhousinginfo@sfgov.org</a>',
```

Comment:
Same question here about nested `t()` calls.

Reply:
Just to clarify, I do think it good practice to _not_ hardcode strings like email addresses. However, you could have to create a new entry into the translation file, something like this:
https://github.com/SFDigitalServices/sf-dahlia-web/blob/4dd1d9bc8f42c2b57c2f099afee0e213dd003112/app/assets/json/translations/react/en.json#L46